in-progress/7482-sequencer-prover-test-net.md

```diff
@@ -0,0 +1,409 @@
+- [x] have a mechanism to incentivize "proposers" to participate in the "pending chain"
+- [x] have a CI/CD framework to easily deploy the network in different configurations for modeling, stress and regression tests
+- [x] demonstrate building of full, proven rollup blocks on L1
+- [x] have a design that can support forced inclusions
+- [ ] support forced inclusions
+- [ ] be able to select/rotate the set of "proposers" permissionlessly
+- [ ] allow permissionless participation from "provers"
+- [ ] a governance mechanism to support upgrades and integration with execution environment
+- [ ] have an enshrined mechanism to punish "proposers" for misbehaving, and a quantifiable guarantee of the punishment
+- [ ] do not depend on the "pending chain" for liveness
+- [ ] do not require a hard fork to take advantage of most software updates
+- [ ] integrated with the execution environment
+
+Thus, SPRTN v1.0.0 is effectively a stepping stone toward MainNet.
+
+In a sense, part of SPRTN v1.0.0 is a subset of [Option B-](https://hackmd.io/B1Ae2zp6QxaSDOQgdFKJkQ#Option-B-), with key differences:
+- the validator set is small and defined by an Aztec Labs multisig (no PoS)
+- all validators re-execute transactions
+- TxEffects are committed to DA as opposed to TxObjects
+- no built-in slashing
+- no forced inclusions
+- no optimistic signature verification
+- no fallback to direct L1 sequencing (i.e. based sequencing)
+
+**However, as stated, our design and implementation will be forward-compatible with the remaining MainNet requirements, and subsequent milestones will satisfy requirements based on their priority.**
+
+To that end, there is a separate issue for [outlining MainNet](https://github.com/AztecProtocol/engineering-designs/pull/11).
+
+## Introduction
+
+We now introduce the broad design of the SPRTN v1.0.0.
+
+### SPRTN v1.0.0 Definitions
+
+**Validator**
+A node that is participating in consensus by producing blocks and attesting to blocks
+
+**Prover**
+A node that is responsible for generating proofs for blocks/epochs.
+
+**Slot**
+Time is divided into fixed length slots. Within each slot, exactly one validator is selected to propose a block. A slot might be empty if no block is proposed. Sometimes the validator selected to propose is called proposer or sequencer
+
+**Epoch**
+A fixed-length sequence of slots
+
+**Committee**
+A list of validators to propose/attest blocks for an epoch. The committee is stable throughout the epoch. Attesters are active throughout the entire duration, and one proposer per slot is active.
+
+**Attestation**
+A vote on the head on the chain.
+
+**Confirmed**
+Of a block, if it is included in a chain.
+
+**Confirmation Rules**
+Of a chain, the set of conditions that must be met for a block to be considered confirmed.
+
+
+### SPRTN v1.0.0 Chains
+
+We will explicitly support multiple concurrent "chains":
+
+- "Pending Chain" - Reflects blocks published to L1 with their state diffs, but not yet been proven.
```

Comment:
Not a fan of the "Pending" name, since it [already has semantics in Ethereum](https://docs.chainstack.com/docs/understanding-pending-block-on-ethereum) which differ from these.

Reply:
Let us know if you have a better option!